src/commands/pickFuncProcess.ts

```diff
@@ -47,27 +48,46 @@ export async function pickFuncProcess(actionContext: IActionContext): Promise<st
 }
 
 async function getFuncHostPid(): Promise<string | undefined> {
-    const multipleProcError: Error = new Error(localize('multipleFuncHost', 'Detected multiple processes running the Functions host. Stop all but one process in order to debug.'));
+    let pids: string[] = [];
+    // If the cli is self-contained, the command will look like this: func host start
+    pids = pids.concat(await getMatchingPids(/.*func.*/, /.*host.*start.*/));
```

Comment:
How fast are these awaits?  Should you run them in parallel?

Reply:
They're very fast

Reply:
Can't hurt though